gen/pb-go/flyteidl/service/flyteadmin/api/swagger.yaml

```diff
@@ -3652,6 +3673,22 @@ definitions:
       tags:
       - "tags"
       - "tags"
+  adminExecutionRecoverRequest:
+    type: "object"
+    properties:
+      id:
+        description: "Identifier of the workflow execution to recover."
+        $ref: "#/definitions/coreWorkflowExecutionIdentifier"
+      name:
+        type: "string"
+        title: "User provided value for the recovered execution.\nIf none is provided\
+          \ the system will generate a unique string.\n+optional"
+      parent_node_execution:
+        description: "Which node (if any) launched this execution. This situation\
```

Comment:
when you mean this -> you mean the referenced execution?

Reply:
updated